de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/simulator/JSSimulatorNodeModel.java

```diff
@@ -54,11 +48,15 @@
     extends AbstractWizardNodeModel<JSSimulatorViewRepresentation, JSSimulatorViewValue>
     implements PortObjectHolder {
 
+  private final JSSimulatorConfig m_config = new JSSimulatorConfig();
+
   private static final NodeLogger LOGGER =
       NodeLogger.getLogger("JavaScript FSK Simulation Configurator");
 
   private FskPortObject port;
 
+  private static final ObjectMapper MAPPER = FskPlugin.getDefault().MAPPER104;

```

Comment:
We should have constants together at the top. MAPPER should be defined right after LOGGER. https://www.cs.cornell.edu/courses/JavaAndDS/JavaStyle.html#fielddecs